Login.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, Text, View, Button } from 'react-native';
+import firebaseConfigInfo from './firebaseConfig';
```

Comment:
I think this can be just `import './firebaseConfig'`. My understanding of how this works is that when this Login.js file runs it will go through all the imports and run those files. You don't need the `firebaseConfigInfo` part because you are not using it anywhere in the file.